src/Pages/FieldInfoPage.js

```diff
@@ -222,6 +229,9 @@ function FieldInfoPage() {
             );
           })}
         </Select>
+        {showLocalityError ? (
+          <FormHelperText>Select a locality</FormHelperText>
+        ) : null}
```

Comment:
Can do `showLocalityError && (<FormHelperText>Select a locality</FormHelperText>)` for brevity?

Reply:
done